modules/contact/test/tests/pdass_problems/cylinder_friction_penalty_adaptivity.i

```diff
@@ -31,6 +31,24 @@
   reference_vector = 'ref'
 []
 
+[Variables]
+  [disp_x]
+    order = FIRST
+    family = LAGRANGE
+  []
+  [disp_y]
+    order = FIRST
+    family = LAGRANGE
+  []
+[]
+
+[Physics/SolidMechanics/QuasiStatic/all]
```

Comment:
This needs an additional test with two physics blocks with block restrictions

Reply:
There's already such a test in test/tests/actions/two_block.i . 1 physics on block 1, the other on block 2 (see `tests` file for cli_args)